src/test/java/org/xwiki/git/script/GitScriptServiceTest.java

```diff
@@ -55,6 +56,54 @@
 
     private File testRepository;
 
+    @Unstable
+    private void getGitRepositoryAndFindAuthors(String username, String accessCode) throws Exception
```

Comment:
How can a method named "get*" not return anything? :)

Reply:
Normally you're going to say that this is how the test methods are named :) That's because we're following the test best practices defined at https://dev.xwiki.org/xwiki/bin/view/Community/Testing/JavaUnitTesting/#HBestpractices

Reply:
I saw on the original code it had "get*" on the void so I created just like that.

Reply:
Again there was a reason to have get in a @Test. You need to ask yourself why and understand it. Do you understand why now?

Reply:
From what I perceive: the get* here is not in the same meaning of returning something but rather to test the use case (which is getting that repository from our created Test repo in the `@Before` method).

Reply:
Please let me know if I am wrong about this.